apps/framework-docs/src/pages/moose/streaming/create-stream.mdx

```diff
@@ -65,18 +65,18 @@ export const rawIngestionStream = new IngestPipeline<RawData>("raw_data", {
 
 <Python>
 ```py filename="IngestionStream.py" copy {10}
-from moose_lib import IngestPipeline, Key
+from moose_lib import IngestPipeline, IngestPipelineConfig, Key
 from pydantic import BaseModel
 
 class RawData(BaseModel):
     id: Key[str]
     value: int
 
-raw_ingestion_stream = IngestPipeline[RawData]("raw_data", {
+raw_ingestion_stream = IngestPipeline[RawData]("raw_data", IngestPipelineConfig(
   ingest: True, # Creates an ingestion API endpoint at `POST /ingest/raw_data`
```

Comment:
need " = " instead of ": " --> this is an issue all over this docs page

Reply:
fixed